### ISSUE TITLE
Can't import getMetadata

### ISSUE THREAD

**Issue by gitowiec** [guigrpa/docx-templates#177]
I did also like this
```
import {
  createReport, listCommands,
  // @ts-ignore
  getMetadata,
} from 'docx-templates';
```
But I get
```
TypeError: docx_templates_1.getMetadata is not a function
```

![image](https://user-images.githubusercontent.com/1553195/102611458-3dbd0c00-412f-11eb-9cc5-0ed36bd98d71.png)

**Comment by mathe42** [comment]
This looks like a runtime error not a TypeScript Error. Do you use the latest version of docx-templates?

**Comment by jjhbw** [comment]
My mistake. I see that I forgot to export getMetadata from the index module.

For now you can quickly hack around this using:
```
import { getMetadata } from 'docx-templates/lib/main'
```

I'll push a release with a fix soon.

**Comment by jjhbw** [comment]
Should be fixed in `v4.5.3` which should hit npm soon. Thanks for reporting!
